ehrapy/preprocessing/_encoding.py

```diff
@@ -743,6 +672,7 @@ def _add_categoricals_to_obs(original: AnnData, new: AnnData, categorical_names:
     logger.info(f"The original categorical values `{categorical_names}` were added to obs.")
 
 
+# TODO: Can we delete this and just rely on what is in obs?
```

Comment:
This point is still open to discussion—the last thing that is saved in `adata.uns`. Currently, the original, unencoded columns are stored in `adata.uns`. That information is then used, for example, when reverting encoding. Theoretically, the unencoded columns should also be stored in `adata.obs`, so I assume it would be possible to change the encoding so that it doesn't rely on `adata.uns` at all. 
However, I feel like `obs` is modified by many functions, so I wonder if it's a bit unsafe to rely on the information being stored there until `unencode` is called. I would be curious to know what you @Zethson think - whether I should keep the code as it is or also try to change this last bit stored in `adata.uns`?